packages/datadog-instrumentations/src/jest.js

```diff
@@ -72,6 +72,7 @@ function getWrappedEnvironment (BaseEnvironment) {
       const setNameToParams = (name, params) => { this.nameToParams[name] = params }
 
       if (event.name === 'setup') {
+        this.global._ddtrace = global._ddtrace
```

Comment:
Not sure this is the right way to do this. We might need to update `shimmer` to allow shimming values, but wanted to get thoughts before making a bigger change like that.

Reply:
I think the important thing is to make sure a test is added to cover this and validate the change.

Reply:
Tests are a bit tricky so I've pushed a different PR in #2003. It's your idea though @mdbenjam :)